src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -151,6 +153,12 @@ class SharedLinkSection extends React.Component<Props, State> {
                     onCopyError();
                 });
         }
+
+        if (sharedLink.url === null) {
```

Comment:
Based on the flow type definition for the sharedLink prop, sharedLink.url won't be null but rather an empty string "" when there is no shared link. Also you'll want to check for a change in sharedLink url rather than anytime its empty, otherwise you'll also focus on the toggle when the modal first launches. I recommend moving your toggleRef focus logic up into this block: https://github.com/box/box-ui-elements/blob/master/src/features/unified-share-modal/SharedLinkSection.js#L131-L133